messages/common.proto

```diff
@@ -1,5 +1,9 @@
 syntax = "proto3";
 
+// Types that are shared by all microcontrollers.
+// To avoid cyclic dependencies, we must avoid importing anything else.
+package orb;
```

Comment:
`orb` might be a little too generic
what about `mcu`? this would give `mcu.main` and `mcu.security` and `mcu` for all common messages right? 
because we use protobuf to communicate with the orb relay as well 🤔

Reply:
I did it to avoid conflict with other third party protobufs.
Should we do orb.mcu?

Reply:
works for me